SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -332,6 +340,11 @@ void USpatialReceiver::ProcessRemoveComponent(const Worker_RemoveComponentOp& Op
 	}
 
 	StaticComponentView->OnRemoveComponent(Op);
+
+	if (Op.component_id == SpatialConstants::UNREAL_METADATA_COMPONENT_ID)
```

Comment:
We should do this instead of the *Actor get condition above when processing metadata

Reply:
What do you mean? Do this first and not do the actor get if it is metadata?

Reply:
Yep

Reply:
done